### PR TITLE
fix: dashboard load nft

### PR DESCRIPTION
**Pull request by hanszhoou**
## Description

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by hanszhoou** [comment]
@wenluomask

File: packages/dashboard/src/pages/Wallets/components/CollectibleList/index.tsx [hunk 1]
```diff
@@ -139,7 +139,7 @@ export const CollectibleListUI = memo<CollectibleListUIProps>(
             const width = ref.current.offsetWidth
             const height = ref.current.offsetHeight - 60
             const baseSize = Math.floor(width / ITEM_SIZE.width) * Math.floor(height / ITEM_SIZE.height)
-            setLoadingSize((prev) => prev ?? Math.floor(baseSize * 0.8))
+            setLoadingSize((prev) => prev || Math.max(Math.floor(baseSize * 0.8), 10))
```
Comment (hanszhoou) [inline packages/dashboard/src/pages/Wallets/components/CollectibleList/index.tsx:143]: `0 ?? 20 === 0`, `0 || 20 === 20`, meanwhile ensure load 10 NFTs at least.

File: packages/mask/src/plugins/Wallet/SNSAdaptor/ConnectWalletDialog/ConnectionProgress.tsx [hunk 2]
```diff
@@ -60,44 +60,46 @@ export function ConnectionProgress(props: ConnectionProgressProps) {
...
-                            <Typography className={classes.error} color="red" variant="body2">
-                                {error.message?.includes('Already processing eth_requestAccounts') ||
-                                error.message?.includes(
-                                    "Request of type 'wallet_requestPermissions' already pending for origin",
-                                )
-                                    ? t('plugin_wallet_metamask_error_already_request')
-                                    : 'Error connecting.'}
-                            </Typography>
+        <>
+            <Paper elevation={0}>
```
Comment (zhouhanseng) [inline packages/mask/src/plugins/Wallet/SNSAdaptor/ConnectWalletDialog/ConnectionProgress.tsx:64]: Fix a dashboard ui issue.

File: packages/mask/src/plugins/Wallet/SNSAdaptor/ConnectWalletDialog/ConnectionProgress.tsx [hunk 3]
```diff
@@ -60,44 +60,46 @@ export function ConnectionProgress(props: ConnectionProgressProps) {
...
+                <Card className={`${classes.content} dashboard-style`} elevation={0}>
```
Comment (hanszhoou) [inline packages/mask/src/plugins/Wallet/SNSAdaptor/ConnectWalletDialog/ConnectionProgress.tsx:65]: Fix a dashboard ui issue.

**Review by Lanttcat — DISMISSED** [review]

File: packages/dashboard/src/pages/Wallets/components/CollectibleList/index.tsx (hunk 1 above)
Comment (septs) [reply packages/dashboard/src/pages/Wallets/components/CollectibleList/index.tsx:143]: plz write the comment in code

File: packages/mask/src/plugins/Wallet/SNSAdaptor/ConnectWalletDialog/ConnectionProgress.tsx [hunk 4]
```diff
@@ -60,44 +60,46 @@ export function ConnectionProgress(props: ConnectionProgressProps) {
...
+                        <Box display="flex" flex={1} flexDirection="column" sx={{ marginLeft: 2 }}>
+                            {connected ? (
+                                <Typography>
+                                    {t('plugin_wallet_connected_with')} {resolveProviderName(providerType)}
+                                </Typography>
+                            ) : (
+                                <Typography>
+                                    {t('plugin_wallet_connect_with')} {resolveProviderName(providerType)}
+                                </Typography>
+                            )}
```
Comment (septs) [inline packages/mask/src/plugins/Wallet/SNSAdaptor/ConnectWalletDialog/ConnectionProgress.tsx]:
```suggestion
                                <Typography>
                                    {t(connected ? 'plugin_wallet_connected_with' : 'plugin_wallet_connect_with')} {resolveProviderName(providerType)}
                                </Typography>
```

**Review by septs — APPROVED** [review]

**Review by yanzhihong23 — APPROVED** [review]

**Pull request merged**